### ISSUE TITLE
Texecom2mqtt Locks up for no reason

### ISSUE THREAD

**Issue by brinzlee** [dchesterton/texecom2mqtt-hassio#22]
I’m still getting the following message and the system power and voltage reporting the same values again and again with no mqtt sensor updates on any of the detectors…This happens quite regularly and is only fixed by stopping the texecom2mqtt and restarting…Is there any way to detect this and restart the addon automatically when this condition occurs. I’m not sure if there is any fundamental difference but I am using ComWifi on ComPort 3 of the alarm panel. Can you also tell me what error
Command 25 timed out (attempt 1, id: 86) is from the following log output
Many thanks

```
2021-06-16 10:31:09 - DEBUG: Updating system power...
2021-06-16 10:31:13 - DEBUG: Command 25 timed out (attempt 1, id: 86).
2021-06-16 10:31:16 - INFO: Kitchen PIR status changed to Active
2021-06-16 10:31:16 - DEBUG: Publishing to texecom2mqtt/zone/kitchen_pir: {"name":"Kitchen PIR","number":4,"areas":["A"],"status":1,"type":"Guard"}
2021-06-16 10:31:16 - DEBUG: Command 25 timed out (attempt 2, id: 86).
2021-06-16 10:31:20 - DEBUG: Command 25 timed out (attempt 3, id: 86).
2021-06-16 10:31:23 - DEBUG: Publishing to texecom2mqtt/power: {"battery_charging_current":18,"battery_voltage":14.05,"panel_current":549,"panel_voltage":14.19}
2021-06-16 10:31:59 - DEBUG: Updating system power...
2021-06-16 10:32:00 - DEBUG: Publishing to texecom2mqtt/power: {"battery_charging_current":18,"battery_voltage":14.05,"panel_current":549,"panel_voltage":14.19}
2021-06-16 10:32:49 - DEBUG: Updating system power...
2021-06-16 10:32:50 - DEBUG: Publishing to texecom2mqtt/power: {"battery_charging_current":18,"battery_voltage":14.05,"panel_current":549,"panel_voltage":14.19}
2021-06-16 10:33:39 - DEBUG: Updating system power...
2021-06-16 10:33:40 - DEBUG: Publishing to texecom2mqtt/power: {"battery_charging_current":18,"battery_voltage":14.05,"panel_current":549,"panel_voltage":14.19}
```

**Comment by no1knows** [comment]
+1, same here using a SmartCom. I have to manually restart texecom2mqtt every few days. Everything else working fine.

**Comment by dchesterton** [comment]
I've managed to get to the bottom of this bug and it has _finally_ been fixed in v1.2.0. Closing this.
